server/fileupdater.mjs

```diff
@@ -157,6 +157,9 @@ function v3RemoveComputeAndRandomAndApplyVariables(routine) {
   }
 
   for(const [ i, op ] of Object.entries(routine)) {
+    if(!op || !op.func)
```

Comment:
What is !op accomplishing here?

Reply:
If `op == null`, it would crash otherwise.